cellular: astyle fix

0xc0170 opened the pull request:
### Description

<!-- 
    Required
    Add here detailed changes summary, testing results, dependencies 
    Good example: https://os.mbed.com/docs/latest/reference/workflow.html (Pull request template)
-->

Running astyle for cellular feature. I run it on all files in the folder. If there are any that should be excludeed, let me know, and I'll add it to the astyle ignore file.

### Pull request type

<!-- 
    Required
    Please add only one X to one of the following types. Do not fill multiple types (split the pull request otherwise).
    Please note this is not a GitHub task list, indenting the boxes or changing the format to add a '.' or '*' in front
    of them would change the meaning incorrectly. The only changes to be made are to add a description text under the
    description heading and to add a 'x' to the correct box.
-->
    [X] Fix
    [ ] Refactor
    [ ] New target
    [ ] Feature
    [ ] Breaking change

0xc0170 committed:
cellular: astyle fix

hasnainvirk commented on features/cellular/UNITTESTS/stubs/ATCmdParser.cpp:
```diff
@@ -398,18 +398,18 @@ bool ATCmdParser::process_oob()
         while (oob) {
             if (i == (int)oob->len && memcmp(
-                    oob->prefix, _buffer, oob->len) == 0) {
+                        oob->prefix, _buffer, oob->len) == 0) {
```
This could fit on the same line like
```
if (i == (int)oob->len && memcmp(oob->prefix, _buffer, oob->len) == 0) {
```

hasnainvirk reviewed (approved):
Looks good. Just a minor comment.

hasnainvirk commented on features/cellular/UNITTESTS/at/at_cellularbase/at_cellularbasetest.cpp:
```diff
@@ -20,15 +20,13 @@
 
 TEST_GROUP(AT_CellularBase)
```
For being consistent with function expansion, please change, void setup() { to:
```
void setup()
{
}
```
as it was originally. Martin will be patching astyle script to ignore expanding functions in the pre-processed objects like TEST_GROUP.

0xc0170 replied:
Good catch, I'll check these macros and will need to be ignored not to be changed as they are not seen by formatter (is it a function/class/etc) so its treated as function by default.

0xc0170 committed:
cellular: fix tests that astyle breaks

Ignore TEST_GROUP by AStyle. As this uses macro, is not visible to formatter.
By default, it assumes it is a function and treats methods as blocks inside a
function (inlined {}). We ignore it.

We might just not format UNITTESTS in the future if we face similar issues
in the future.

0xc0170 commented:
All `TEST_GROUP` were fixed

cmonr commented:
/morph build

0xc0170 commented:
@ARMmbed/mbed-os-maintainers Please review
